packages/ag-charts-community/src/chart/interaction/keyNavManager.ts

```diff
@@ -0,0 +1,64 @@
+import { BaseManager } from './baseManager';
+import type {
+    FocusInteractionEvent,
+    InteractionEvent,
+    InteractionManager,
+    KeyInteractionEvent,
+} from './interactionManager';
+
+type KeyNavEventType = 'tab' | 'nav-hori';
+
+export type KeyNavEvent<T extends KeyNavEventType = KeyNavEventType> = {
+    type: T;
+    delta: number;
+    interactionEvent: InteractionEvent;
+};
+
+export class KeyNavManager extends BaseManager<KeyNavEventType, KeyNavEvent> {
+    private isFocused: boolean = false;
+    private readonly destroyFns: (() => void)[] = [];
```

Comment:
Is it worth pushing this up to `BaseManager` as a responsibility?

Reply:
Done. https://github.com/ag-grid/ag-charts/commit/e6e7942a8043e38f0decd154b34dd2e177092212